addons/spreadsheet_account/models/account.py

```diff
@@ -102,7 +93,7 @@ def spreadsheet_move_line_action(self, args):
             "view_mode": "list",
             "views": [[False, "list"]],
             "target": "current",
-            "domain": domain,
+            "domain": list(domain),
```

Comment:
Is it required for rpc?
If so, maybe also https://github.com/odoo-dev/odoo/blob/a406a4b63da0876ac33fb0b6b0833ec8106b6d16/addons/product/models/product_catalog_mixin.py#L26
which may return `Domain` objects now.